src/sqlfluff/rules/L003.py

```diff
@@ -17,13 +17,10 @@
 class Rule_L003(BaseRule):
     """Indentation not consistent with previous lines.
 
-    Note:
-        This rule used to be _"Indentation length is not a multiple
-        of `tab_space_size`"_, but was changed to be much smarter.
```

Comment:
This is am old comment. Gave up going back through git as to exactly how old but it's old. Don't think we need it anymore.

Reply:
I agree. I think it was changed shortly after I got involved with the project 2+ years ago.